projects/libnest2d_find_clipper.cmake

```diff
@@ -0,0 +1,92 @@
+# Find Clipper library (http://www.angusj.com/delphi/clipper.php).
+# The following variables are set
+#
+# CLIPPER_FOUND
+# CLIPPER_INCLUDE_DIRS
+# CLIPPER_LIBRARIES
+#
+# It searches the environment variable $CLIPPER_PATH automatically.
+
+unset(CLIPPER_FOUND CACHE)
+unset(CLIPPER_INCLUDE_DIRS CACHE)
+unset(CLIPPER_LIBRARIES CACHE)
+unset(CLIPPER_LIBRARIES_RELEASE CACHE)
+unset(CLIPPER_LIBRARIES_DEBUG CACHE)
+
+if(CMAKE_BUILD_TYPE MATCHES "(Debug|DEBUG|debug)")
+    set(CLIPPER_BUILD_TYPE DEBUG)
+else()
+    set(CLIPPER_BUILD_TYPE RELEASE)
+endif()
+
+FIND_PATH(CLIPPER_INCLUDE_DIRS clipper.hpp
```

Comment:
This is a patched file, adapted from the original in libnest2d: https://github.com/tamasmeszaros/libnest2d/blob/master/cmake_modules/FindClipper.cmake

The only modification is the addition of `${CMAKE_INSTALL_PREFIX}` as search path on line 31, 32, 50 and 51. I think it would be prudent to leave the file unmodified, not adjust their code style.

I would love to have used a unified diff file for this instead. That would make the change much more clear. However there doesn't seem to be an easy way to apply this on Windows.